src/library/Navigation/themes.js

```diff
@@ -0,0 +1,115 @@
+/* @flow */
+import { pxToEm } from '../styles';
+
+import type {
+  PrimaryNavThemeFn,
+  SecondaryNavThemeFn,
+  NavigationThemeFn,
+  PrimaryNavItemThemeFn,
+  SecondaryNavItemThemeFn,
+  NavItemThemeFn
+} from './types';
+
+export const primaryNavTheme: PrimaryNavThemeFn = (baseTheme: Object) => ({
+  PrimaryNav_backgroundColor: baseTheme.backgroundColor_themePrimary,
+  PrimaryNav_backgroundColor_minimal: baseTheme.backgroundColor,
+  PrimaryNav_gutter: baseTheme.space_inline_xs,
+  PrimaryNav_paddingHorizontal: baseTheme.space_inline_xl,
+  PrimaryNav_paddingVertical: baseTheme.space_stack_sm
+});
+
+export const secondaryNavTheme: SecondaryNavThemeFn = (baseTheme: Object) => ({
+  SecondaryNav_backgroundColor_pills: baseTheme.backgroundColor,
+  SecondaryNav_backgroundColor_tabs: baseTheme.color_gray_20,
+  SecondaryNav_border: `1px solid ${baseTheme.color_gray_30}`,
+  SecondaryNav_gutter: baseTheme.space_inline_xxs,
+  SecondaryNav_paddingHorizontal: pxToEm(12),
+  SecondaryNav_paddingVertical: baseTheme.space_stack_sm
+});
+
+export const navigationTheme: NavigationThemeFn = (baseTheme: Object) => ({
+  ...primaryNavTheme(baseTheme),
+  ...secondaryNavTheme(baseTheme),
+
+  ...baseTheme
+});
+
+export const primaryNavItemTheme: PrimaryNavItemThemeFn = (
+  baseTheme: Object
+) => ({
+  PrimaryNavItem_paddingHorizontal: baseTheme.space_inset_sm,
+  PrimaryNavItemIcon_color: 'inherit',
+
+  PrimaryNavItem_backgroundColor: 'transparent',
+  PrimaryNavItem_backgroundColor_active: baseTheme.color_theme_80,
+  PrimaryNavItem_backgroundColor_focus: 'transparent',
+  PrimaryNavItem_backgroundColor_hover: baseTheme.color_theme_70,
+  PrimaryNavItem_backgroundColor_selected: baseTheme.color_theme_80,
+  PrimaryNavItem_borderColor: 'transparent',
+  PrimaryNavItem_borderColor_active: 'transparent',
+  PrimaryNavItem_borderColor_focus: baseTheme.color_theme_20,
+  PrimaryNavItem_borderColor_hover: 'transparent',
+  PrimaryNavItem_borderColor_selected: baseTheme.color_theme_80,
+  PrimaryNavItem_color: baseTheme.color_themePrimary,
+  PrimaryNavItem_color_disabled: baseTheme.color_theme_40,
+  PrimaryNavItem_color_hover: baseTheme.color_themePrimary,
+  PrimaryNavItem_color_selected: baseTheme.color_themePrimary,
+
+  PrimaryNavItem_backgroundColor_minimal: 'transparent',
```

Comment:
I realize that you have ordered/grouped these keys intentionally.  It does affect the order that they are displayed in the theme variables table on the website too.  Is this your intent as well, or do you think we should sort that data prior to rendering?

Reply:
Good point. I re-ordered them to be alpha-sorted, like the other component themes.